metadata/src/main/java/org/apache/kafka/controller/QuorumFeatures.java

```diff
@@ -61,6 +62,12 @@ public static Map<String, VersionRange> defaultFeatureMap(boolean enableUnstable
                 enableUnstable ?
                     MetadataVersion.latestTesting().featureLevel() :
                     MetadataVersion.latestProduction().featureLevel()));
+        for (Features feature : Features.PRODUCTION_FEATURES) {
+            features.put(feature.featureName(), VersionRange.of(
+                0,
+                feature.latestProduction()
```

Comment:
Should this take `enableUnstable` into consideration?

Reply:
This is in the followup. :) It changes a ton of files so I would prefer to do it separately.